packages/embed/src/setups/version3.js

```diff
@@ -17,7 +17,7 @@ export async function getBingSpeechToken(directLineToken, bingSpeechTokenURL) {
 export default async function setupVersionFamily3(
   { assets },
   { botId, directLineURL: domain, speechTokenURL, userId, webSocket },
-  { language, secret, token, username }
+  { language, secret, token, username, conversationStartLocale }
```

Comment:
You don't need to touch this file. This is for Web Chat v3, which do not support the "Start Conversation" locale.